src/styles/blocks/catalog.scss

```diff
@@ -18,6 +18,14 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: $duration;
+
+    &:hover {
+      transition: $duration;
+      transform: perspective(100px) translateZ(-20px);
```

Comment:
Read the task carefully:
_Smoothly (during 300ms) increase the card by 20 percent on hover (neighboring cards should not be affected)_

You did it wrong.